assets/music/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "menu": "free.mp3.wav",
```

Comment:
Is this supposed to be .mp3.wav?

Reply:
the song name is called free.mp3, but this is a wav version of it lol